actions/rules/action_field_set.go

```diff
@@ -3,12 +3,12 @@ package rules
 import (
 	"context"
 	"encoding/json"
-	"errors"
-	"github.com/fabric8-services/fabric8-wit/application"
 	"reflect"
 
+	"github.com/pkg/errors"
```

Comment:
We have the unwritten convention to import this package as `errs` to avoid confusion with our own `errors` package and the standard `errors` package.

Reply:
Done.